ticker_service.go

```diff
@@ -26,14 +29,15 @@ func (ts *TickerService) Start() error {
 
 	devices, err := ts.ds.GetAll(0, 0)
 	if err != nil {
+		log.Print(err)
 		return errors.New("failed to start measurements sending")
 	}
 
 	ts.stop = make(chan bool)
 	ts.isRunning = true
 
 	for _, device := range devices {
-		ts.createTickerForDevice(device)
+		go ts.createTickerForDevice(device)
```

Comment:
Your case will work fine but for future reference - be careful while using closures:

https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables